prow/config/inrepoconfig.go

```diff
@@ -74,7 +74,7 @@ func defaultProwYAMLGetter(
 
 	mergeMethod := c.Tide.MergeMethod(organization, repository)
 	log.Debugf("Using merge strategy %q.", mergeMethod)
-	if err := repo.MergeAndCheckout(baseSHA, mergeMethod, headSHAs...); err != nil {
+	if err := repo.MergeAndCheckout(baseSHA, string(mergeMethod), headSHAs...); err != nil {
```

Comment:
Why was the `mergeMethod` changed to be a generic string rather than a typed one?

Reply:
IIRC this kept the dependency tree much more sane and also may have helped a circular dependency?

Reply:
Also GitHub API merge methods != merge methods in git...